src/Utils.php

```diff
@@ -13,9 +13,7 @@ function get_pointer(string $current, string ...$parts): string {
 
 function get_function_name_from_function(mixed $function): string {
   $func_name = '';
-
-  if (\HHVM_VERSION_ID >= 42100) {
```

Comment:
Is there an actual behavior change here? Every version after 4.21 has a version ID > 42100 - just that version 4.47 and later will 'look like' version 40.47, which is fine for the purpose of this constraint

Reply:
based on https://github.com/slackhq/hack-json-schema/pull/37, perhaps the check should be entirely removed instead?